frontend/src/views/Table.js

```diff
@@ -17,7 +17,11 @@ const Table = () => {
     // state.totalProducts ? <Cart /> : null,
     state.orders.length
       ? state.orders.map((order, orderNr) => (
-          <Order key={order._id} orderNr={orderNr} order={order} />
+          <Order
+            key={order._id}
+            orderNr={state.orders.length - orderNr}
+            order={order}
+          />
         ))
       : null,
     !state.orders.length && !state.totalProducts ? (
```

Comment:
The code change looks fine to me. The only improvement suggestion I would make is to consider adding more code comments to increase code readability and make it easier for other developers to understand the purpose of each line of code.